sw/device/lib/dif/dif_clkmgr.c

```diff
@@ -42,6 +42,35 @@ static bool clkmgr_measure_ctrl_regwen(const dif_clkmgr_t *clkmgr) {
                              CLKMGR_MEASURE_CTRL_REGWEN_EN_BIT);
 }
 
+/**
+ * Checks if the jitter enable register is locked.
+ *
+ * Jitter enable register is locked by CLKMGR_JITTER_REGWEN.
```

Comment:
I think this should probably be "The jitter enable ..." ? (Similarly for "The external clock ..." below)